Haya_AlMousa/week_02/day_03/aboutme.js

```diff
@@ -0,0 +1,11 @@
+
+
+var body = document.querySelector("body");
+var styles = getComputedStyle(body);
+console.log(styles.fontFamily);
+body.style.fontFamily = "Arial, sans-serif";
+
+var span = document.querySelector("#nickname");
+nickname.setAttribute("Haya");
```

Comment:
Please come up to me in class, I'll explain the steps of setting attributes of DOM nodes.